src/providers/microsoft/crypto/mod.rs

```diff
@@ -60,15 +60,15 @@ pub fn p12_to_ssh_pubkey(p12_der: &[u8]) -> Result<PublicKey> {
     // PKCS12 has the ability to have a password, but we don't have one, hence
     // empty string.
     let p12 = Pkcs12::from_der(p12_der).context("failed to get pkcs12 blob from der")?;
-    let p12 = p12.parse("").context("failed to parse pkcs12 blob")?;
+    let p12 = p12.parse2("").context("failed to parse pkcs12 blob")?;
 
-    // PKCS12 has three parts: a pkey, a main x509 cert, and a list of other
-    // x509 certs. The list of other x509 certs is called the chain. There is
-    // only one cert in this chain, and it is the ssh public key.
-    let chain = p12
-        .chain
+    // ParsedPKCS12_2 has three parts: a pkey, a main x509 cert, and a list of other
+    // x509 certs. The list of other x509 certs is called the ca. There is
+    // only one cert in this ca, and it is the ssh public key.
```

Comment:
Hmm.  This was originally phrased as an implementation-neutral description of PKCS12 (extremely simplified), and is now trying to be a description of the struct, using a confusing mix of terminology ("only one cert in this ca").  Let's stick with the generic approach but add a caveat, e.g. `The list of other x509 certs is called the chain (here stored in the 'ca' field)...`

Reply:
Is it not actually called a `ca chain`? if it is does this sound better 

``` // ParsedPKCS12_2 has three parts: a pkey, a main x509 cert, and a list of other
    // x509 certs. The list of other x509 certs is called the `ca chain` currently denoted as `ca`. There is
    // only one cert in this `ca chain`, and it is the ssh public key.
    
```

Reply:
"CA chain" is somewhat better but I think "certificate chain" is more correct.

Reply:
Ah, ok I went ahead and went for `certificate chain`. I think whats noted is now better, and makes sense. LMK if you disagree.